bugsnag/django/__init__.py

```diff
@@ -22,9 +22,8 @@ def add_django_request_to_notification(notification):
 
     if hasattr(request, 'user') and request.user.is_authenticated():
         try:
-            name = " ".join([request.user.first_name or '',
-                             request.user.last_name or ''])
-            notification.set_user(id=request.user.username,
+            name = request.user.get_full_name()
+            notification.set_user(id=request.user.get_username(),
                                   email=request.user.email, name=name)
```

Comment:
Also, according to this documentation, the `USERNAME_FIELD` might be a foreign key as per:

> USERNAME_FIELD now supports ForeignKeys. Since there is no way to pass model instances during the createsuperuser prompt, expect the user to enter the value of to_field value (the primary_key by default) of an existing instance.

and:

> `get_username():`
>     Returns the value of the field nominated by USERNAME_FIELD.

Which could mean the username is also not a string, unsure if this causes any problems, just wanted to point out that it might not be a string.